docker-compose.yml

```diff
@@ -167,15 +178,10 @@ services:
     environment:
       - USER=${PECAN_RSTUDIO_USER:-carya}
       - PASSWORD=${PECAN_RSTUDIO_PASS:-ilinois}
-    depends_on:
```

Comment:
Should we depend upon rstudio-nginx?

Reply:
mmmm, maybe, it will start fine without it, so there is no need for you to wait for it to be up.